make/make.py

```diff
@@ -256,14 +229,16 @@ def help():
 
 
 def main(command="help", *args):
+    # pylint: disable=keyword-arg-before-vararg
+    # This whole system will go away, ignore this pylint warning
     """
     Dispatch on command name, passing all remaining parameters to the
     module-level function.
     """
     try:
         function = _exported[command]
     except KeyError:
-        raise RuntimeError("No such command: %s" % command)
+        raise RuntimeError("No such command: %s" % command) from None
```

Comment:
should probably be `except KeyError as e: ... raise ... from e`

Reply:
Yeah, I went cheap on those errors because I wasn't sure if any of that was going to survive when made this whole repo look more like a library repo. I'll re-work that when I do the github action work.